platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -61,6 +62,8 @@ public Object execute() {
           (StringUtils.isEmpty(exportDirectoryArgument)
               ? defaultExportDirectory
               : Paths.get(exportDirectoryArgument));
+      SecurityLogger.audit(
```

Comment:
These commands should be getting audited already with the command listener that I created a while back. Can you check to see if we have 2 audits for this command now.

Reply:
I had copied that audit from another command (don't remember which one). I'll remove it.